tableOfContents.md

```diff
@@ -13,3 +13,6 @@ order: 3
 1. [Preface](/detel-book/chapter/introduction/preface/)
 
 2. [My New Chapter](/detel-book/chapter/myChapter/)
+
+3. [Design-based Research in TEL: an introduction](/detel-book/chapters/Lorena_UAveiro/DBR/)
+4. [Creating Open Educational Resources with the H5P tool](/detel-book/chapters/Lorena_UAveiro/OER/)

```

Comment:
Could you propose a structure for the table of contents that matches with the one of the current live version? https://github.com/ea-tel/detel-book/blob/master/tableOfContents.md

Reply:
I will make a structure and update both in the TOC and the chapters.

Reply:
structure is already there. I will adapt to it.